Server/ConcordiaCurriculumManager/Services/DossierService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using ConcordiaCurriculumManager.DTO.Courses;
+using ConcordiaCurriculumManager.DTO.Dossiers;
+using ConcordiaCurriculumManager.Models.Curriculum;
+using ConcordiaCurriculumManager.Models.Curriculum.Dossiers;
+using ConcordiaCurriculumManager.Repositories;
+using System;
+using System.Security.Claims;
+
+namespace ConcordiaCurriculumManager.Services;
+
+public interface IDossierService
+{
+    public Task<List<Dossier>> GetDossiersByID(Guid ID);
+}
+
+public class DossierService : IDossierService
+{
+    private readonly ILogger<DossierService> _logger;

```

Comment:
Can you remove all these unused class attributes?

Reply:
Sounds good